docs/en/sql-reference/statements/show.md

```diff
@@ -6,14 +6,16 @@ sidebar_label: SHOW
 
 # SHOW Statements
 
-## SHOW CREATE TABLE
+## SHOW CREATE TABLE or DATABASE
 
 ``` sql
-SHOW CREATE [TEMPORARY] [TABLE|DICTIONARY|VIEW] [db.]table|view [INTO OUTFILE filename] [FORMAT format]
+SHOW [CREATE] [TEMPORARY] [TABLE|DICTIONARY|VIEW|DATABASE] [db.]table|view [INTO OUTFILE filename] [FORMAT format]
 ```
 
 Returns a single `String`-type ‘statement’ column, which contains a single value – the `CREATE` query used for creating the specified object.
 
+`SHOW TABLE t`, `SHOW CREATE t` and `SHOW DATABASE db` have the same meaning as `SHOW CREATE TABLE|DATABASE t|db`, but `SHOW t` and `SHOW db` are not supported.
```

Comment:
Kind of unrelated, but perhaps you could shorten l. 17 to:

"Returns a single column of type `String` containing the `CREATE` query used for creating the specified object."

Reply:
But it's almost same as l. 15.

Reply:
My fault, I meant the suggestion as a replacement for l. 15.